globus_cli/services/transfer/ls.py

```diff
@@ -46,11 +50,11 @@ def _get_ls_res(client, path, endpoint_id, recursive, depth):
 
     if depth >= 0:
         # do an `ls` call against current path, use it to seed the result_doc
-        res = client.operation_ls(endpoint_id, path=path)
+        res = client.operation_ls(endpoint_id, **ls_kwargs)
         result_doc['DATA'] = [item for item in res]
         # set path in order to get trailing slash, if necessary
         result_doc['path'] = res['path']
-        path = res['path']
+        path = ls_kwargs['path'] = res['path']
```

Comment:
It's not necessary, but I added it before realizing that.

It seemed fine to leave in, but maybe it's confusing -- would you like me to amend and remove it?